cirq/testing/circuit_compare.py

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 
 from typing import Tuple, Sequence, TYPE_CHECKING
+try:
```

Comment:
This is far too hacky of a way to do this IMO.

I would recommend doing the following:

`import itertools`

use `itertools.zip_longest`

in `python2.7-generate.sh`, add a loop that search-replaces `itertools.zip_longest` with `itertools.izip_longest`. (Mimic the loop that does the `__str__` -> `__unicode__` translation)